### PR TITLE
Add recipe for natu

### PR DESCRIPTION
**Pull request by johnflavin**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/natu/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - natu
+
+about:
+  home: http://kdavies4.github.io/natu/index.html
+  license_file: LICENSE.txt
```
Comment (scopatz) [inline recipes/natu/meta.yaml]: Please add license and license_family entries

Comment (johnflavin) [reply recipes/natu/meta.yaml]: I didn't know what to include for those. The license text doesn't make reference to any particular named license or license family that I know of. https://github.com/kdavies4/natu/blob/master/LICENSE.txt

Comment (scopatz) [reply recipes/natu/meta.yaml]: This looks like a modified BSD license.  However, the last clause makes me think we can't redistribute this:

```
This software is controlled under the jurisdiction of the United States
      Department of Commerce and subject to Export Administration Regulations.
      By downloading or using the Software, you are agreeing to comply with
      U. S. export controls.  Diversion contrary to law is prohibited.  The
      software cannot be exported or reexported to sanctioned countries that
      are controlled for Anti-Terrorism (15 CFR Part 738 Supplement 1) or to
      denied parties
```

CC @conda-forge/core This is something that would be have been caught by https://github.com/conda-forge/conda-smithy/pull/1256

**Comment by isuruf** [comment]
@conda-forge-admin, lint

File: recipes/natu/meta.yaml (hunk 1 above)
Comment (johnflavin) [reply recipes/natu/meta.yaml]: I have added `BSD` for both `license` and `license_family`. Leaving aside for the moment the issue of whether this package is redistributable, are these the correct values? Is there some way to indicate in the `license` value that the license is a _modified_ BSD, and not just BSD?

**Comment by johnflavin** [comment]
I asked the author of `natu` about the potential for modifying the license in https://github.com/kdavies4/natu/issues/36. But given that this ticket was opened years ago, and the author responded that he didn't have time to work on it, I have little hope that the situation will be addressed. I'll close this PR.

**Pull request closed without merging**